src/test/java/org/folio/linked/data/integration/DataImportEventListenerIT.java

```diff
@@ -109,6 +117,8 @@ void shouldConsumeInstanceCreatedEventFromDataImport() {
       assertThat(edge.getTarget()).isNotNull();
       assertThat(edge.getPredicate()).isNotNull();
     });
+
+    assertWorkIsIndexed(result);
```

Comment:
I wanted to do a similar assertion that no message was sent to Inventory. Do you know how to check No message was sent?

Reply:
As I understand, you can use approach of test-listeners in our test-code database
for example, `org.folio.linked.data.test.kafka.KafkaInventoryTopicListener`
but I'm not sure that is the best option for a negative scenario (when the message was sent)

Reply:
one of the way inject stab bean of Kafka,  sender (wich using real one) instead of a real one and checks the interaction of this stab

Reply:
Done